### PR TITLE
INT-310 echo success

### PR DESCRIPTION
**Pull request by mburns**

### CODE REVIEW

File: preflight.sh [hunk 1]
```diff
@@ -76,3 +76,5 @@ gcloud compute firewall-rules create "$FIREWALL_RULE" \
 gcloud storage buckets create "$DEP_BUCKET" \
 	--project="$PROJECT_ID" \
 	--location="$REGION"
+
+printf "Setup script was \033[0;32msuccessful\033[0m\n"
```
Comment (johnlk) [inline preflight.sh:80]: Looks good. We could consider coloring the whole line green. But your call

Comment (johnlk) [reply preflight.sh:80]: Could also define a display success function that'll print out green lines and call it throughout the process

Comment (mburns) [reply preflight.sh:80]: > Could also define a display success function that'll print out green lines and call it throughout the process

ya, this is what I'm leaning towards. Will take a bit more bash-fu to do that.

Comment (johnlk) [reply preflight.sh:80]: Just did something similar for Robin https://github.com/Integral-Healthcare/robin-ai-reviewer/blob/cec79fccc21d66b2d603b5b5dbfe1bb3cd7009e2/src/utils.sh#L4

File: postflight.sh [hunk 2]
```diff
@@ -28,3 +28,5 @@ gcloud compute networks subnets delete "$SUBNET_NAME" \
 
 gcloud compute networks delete "$NET_NAME" \
 	--project="$PROJECT_ID"
+
+printf "\033[0;32msCleanup script was uccessful\033[0m\n"
```
Comment (johnlk) [inline postflight.sh:32]: @mburns "uccessful"

**Pull request merged**